src/acceptor.rs

```diff
@@ -426,6 +452,66 @@ impl rustls_accepted {
     }
 }
 
+/// Represents a TLS alert resulting from accepting a client.
+pub struct rustls_accepted_alert {
+    _private: [u8; 0],
+}
+
+impl Castable for rustls_accepted_alert {
+    type Ownership = OwnershipBox;
+    type RustType = AcceptedAlert;
+}
+
+impl rustls_accepted_alert {
+    /// Write some TLS bytes (e.g. alerts) to the network. The actual network I/O is
```

Comment:
small nit: there should be zero or one alert. if there are more than that, it would be a bug in rustls.
```suggestion
    /// Write some TLS bytes (an alert) to the network. The actual network I/O is
```

Reply:
Fixed up. Thanks :+1: